acl/src/main/java/org/apache/rocketmq/acl/common/AclUtils.java

```diff
@@ -146,13 +148,31 @@ public static boolean isMinus(String minus) {
         }
     }
 
+    public static boolean writeDataObject(String path, Map<String,Object> dataMap) {
+        Yaml yaml = new Yaml();
+        PrintWriter pw = null;
+        try {
+            pw = new PrintWriter(new FileWriter(path));
```

Comment:
Should this "new FileWriter(path)" handler be closed after use?

Reply:
I think it will be ok if the printwriter object has been closed.

Reply:
I look into the implementation of PrintWriter close method,it can close Writer object which is as constructor of paramter(FileWriter)!